### ISSUE TITLE
The Preview button does not indicate if it is a submenu and does not expose its expanded or collapsed state to the screen reader

### ISSUE THREAD

**Issue by puja-iaccessible** [adobe/helix-sidekick-extension#253]
**Description**
The Preview button is not announced as a dropdown list/menu by the screen reader.
There is no way for a screen reader user to distinguish between the parent "preview" button and the "preview" button appearing in the submenu.
The user cannot tell if the preview menu is currently expanded or collapsed.
The user cannot tell how many items are present in the preview menu as they go down the list. There is no way to tell when the submenu ends and when the list of parent menu items is continued.

**To Reproduce**
Steps to reproduce the behavior (using screen reader - NVDA):

1. Open the URL https://main--franklinsidekickrepo--puja-iaccessible.hlx.page/
2. Press Alt key to go to the chrome menu and press tab to go to the sidekick extension 
3. Press enter on the sidekick extension
4. Press tab to go to the preview button and press enter to expand
5. Notice output of screen reader

**Expected behavior**
The role of the preview button should be either "menu" or "dropdown" because it brings up a menu of other buttons. The state of the preview menu should indicate if the menu is currently expanded or collapsed. Once expanded, the submenu items should be a part of a list so that a screen reader user can determine how many menu items are present inside the "preview" menu. In its current state, there is no distinction for a screen reader user between the top-level preview button and the preview button inside the submenu and the user cannot tell if the menu is currently expanded or collapsed.
The screen reader should announce the following
1. preview as dropdown menu/list
2. the state of the menu should also be announced whether "collapsed" or "expanded"
3. The submenu items should be in a "list" element. 

**Screenshots**
Attached is the screenshot.

**Additional context**
https://www.w3.org/TR/WCAG21/#name-role-value

**Why it matters**
Providing the appropriate role and state to various controls allows screen reader and other assistive technology users to understand and operate those controls appropriately.

![Preview button dropdown 4](https://user-images.githubusercontent.com/123964084/219017324-d83b8d97-28a2-434b-a7dc-6a3e281fffa0.png)

**Comment by rofe** [comment]
@puja-iaccessible is it sufficient to add aria-expanded attributes (true or false) to the dropdowns?

**Comment by rofe** [comment]
Duplicate of #254
